### PR TITLE
Automatic groups

### PR DESCRIPTION
**Pull request by the-glu**

### CODE REVIEW

File: sentry_auth_gitlab/views.py [hunk 1]
```diff
@@ -4,10 +4,61 @@
...
 
 class FetchUser(AuthView):
     def handle(self, request, helper):
         access_token = helper.fetch_state('data')['access_token']
         user = GitLabClient().get_user(access_token)
         helper.bind_state('user', user)
         return helper.next_step()
+
+
+class SetTeams(AuthView):
```
Comment (brukau) [inline sentry_auth_gitlab/views.py]: too many blank lines below, after each of and for it is not necessary

Comment (brukau) [reply sentry_auth_gitlab/views.py]: general comment will be good to have here

**Comment by brukau** [comment]
I see now that potential issue with this approach is that groups are synchronized with teams at the moment of user creation, so if after user has joined new group in GitLab it will not be reflected in Sentry, right?

Ok it is for every login, then it is fine

File: sentry_auth_gitlab/views.py (hunk 1 above)
Comment (the-glu) [reply sentry_auth_gitlab/views.py]: Fixed. Also added some remarks in the readme.

**Review by brukau — APPROVED** [review]

**Pull request merged**